### PR TITLE
Limit software startup size

### PR DESCRIPTION
**Pull request by BeyondXinXin**

### CODE REVIEW

**Comment by BeyondXinXin** [comment]
I have two screens: 1920 * 1080 (main screen) and 3840 * 2160 (secondary screen)

When I maximize and close the software on the secondary screen, the display exception will appear next time I start the software

**Comment by juzzlin** [comment]
I guess this happens on Windows?

File: src/main_window.cpp [hunk 1]
```diff
@@ -1,4 +1,4 @@
-// This file is part of Heimer.
+﻿// This file is part of Heimer.
```
Comment (juzzlin) [inline src/main_window.cpp:1]: Needless change here

**Comment by BeyondXinXin** [comment]
Yes, it appears on windows

**Comment by juzzlin** [comment]
Merged from CLI, thanks!

**Pull request closed without merging**